src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -114,12 +114,12 @@ EditorWidgetBase {
 
   function getPictureFilePath() {
     var evaluatedFilepath = expressionEvaluator.evaluate()
+    var filepath = ( evaluatedFilepath && FileUtils.fileSuffix(evaluatedFilepath) !== '' )
+      ? evaluatedFilepath
+      : ('DCIM/JPEG_' + (new Date()).toISOString().replace(/[^0-9]/g, '') + '.jpg')
+    filepath = filepath.replace('\\', '\/')
```

Comment:
```suggestion
    filepath = filepath.replace('\\', '/')
```

no? you're the js expert

Reply:
both are fine, in js `\` char in strings escapes whatever is afterwards.

Reply:
I removed the escaping char anyways.